pwscf/v6.3/pw2wannier90.f90

```diff
@@ -231,8 +232,9 @@ PROGRAM pw2wannier90
   !
   IF (noncolin.and.gamma_only) CALL errore('pw2wannier90',&
        'Non-collinear and gamma_only not implemented',1)
-  IF (noncolin.and.scdm_proj) CALL errore('pw2wannier90',&
-       'Non-collinear and SCDM not implemented',1)
+! jml
```

Comment:
Be bold - just delete these lines! (we have the history in git)

Reply:
I have done so :) (for both v6.3 and v6.4)